modules/dcache/src/main/java/org/dcache/services/httpd/HttpdRequestLog.java

```diff
@@ -27,11 +28,23 @@
 class HttpdRequestLog extends AbstractLifeCycle
     implements RequestLog
 {
-    private static final Logger LOGGER
-            = LoggerFactory.getLogger(HttpdRequestLog.class);
+    //private static final Logger LOGGER = LoggerFactory.getLogger(HttpdRequestLog.class);
+
+
+    private static final Logger ACCESS_LOGGER = LoggerFactory.getLogger("org.dcache.services.httpd");
```

Comment:
Suggest to keep the previous name of the logger object

e.g. ACCESS_LOGGER   to   LOG or LOGGER